index.bs

```diff
@@ -2914,8 +2925,11 @@ To <dfn noexport>trigger attribution</dfn> given an [=attribution trigger=] |tri
 To <dfn>check whether a moment falls within a window</dfn> given a [=moment=] |moment| and
 a [=report window=] |window|:
 
-1. Return whether |moment| is greater than or equal to [=report window/start=]
-    and |moment| is less than |window|'s [=report window/end=].
+1. If |moment| is less than |window|'s [=report window/start=], return
```

Comment:
optional: What do you think about some alternatives like `falls before`, `falls within`, `falls after`? I think `passed` and `active` are a bit ambiguous, especially `passed` which could indicate success.

Reply:
I was trying to make the names match the debug report strings, but I agree these are easier to understand. Done.